test/api_test.js

```diff
@@ -82,7 +82,7 @@ suite('api', () => {
 
   setup(async () => {
     github = await helper.load('github');
-    github.inst(9090).setRepositories('coolRepo', 'anotherCoolRepo', 'awesomeRepo');
+    github.inst(9090).setRepositories('coolRepo', 'anotherCoolRepo', 'awesomeRepo', 'nonTCGHRepo');
```

Comment:
(a nit: it's not necessarily non tc-gh repo. As in tc-cli case, when tc is not used for pushes)